lib/class-wp-rest-image-editor-controller.php

```diff
@@ -120,16 +119,26 @@ public function apply_edits( $request ) {
 			return new WP_Error( 'rest_unknown_attachment', $error, array( 'status' => 404 ) );
 		}
 
+		$supported_types = array( 'image/jpeg', 'image/png', 'image/gif' );
+		$mime_type       = get_post_mime_type( $request['id'] );
+		if ( ! in_array( $mime_type, $supported_types, true ) ) {
+			return new WP_Error(
+				'rest_cannot_edit_file_type',
+				__( 'Sorry, you are not allowed to edit file type.', 'gutenberg' ),
```

Comment:
```suggestion
				__( 'This type of images cannot be edited.', 'gutenberg' ),
```

Another nitpick :) 
Maybe better than telling the users they are "not allowed to edit".

Reply:
Maybe `This type of file cannot be edited`? Since it isn't necessarily an image?

Reply:
Updated.